Userland/Utilities/tail.cpp

```diff
@@ -87,12 +87,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.add_positional_argument(file, "File path", "file");
     args_parser.parse(arguments);
 
-    auto f = Core::File::construct(file);
-    if (!f->open(Core::OpenMode::ReadOnly)) {
-        warnln("Failed to open {}: {}", f->name(), f->error_string());
-        exit(1);
-    }
-
+    auto f = TRY(Core::File::open(file, Core::OpenMode::ReadOnly));
```

Comment:
IMO this should be squashed into the `tail: Port to LibMain` commit. At least that's how I have done it with the previous Port to LibMain work, trying to convert as much of the existing code to use TRYable funcions.